roles/users/templates/sudoers.d.j2

```diff
@@ -0,0 +1,5 @@
+# {{ ansible_managed }}
+
+{% for service in web_sudoers %}
+{{ web_user }} ALL=(ROOT) NOPASSWD: {{ service }}
+{% endfor %}
```

Comment:
Ah didn't run into that since I've been mostly testing deploys lately with `vagrant` user which already has sudoers access. Fixed.